### ISSUE TITLE
MongoDB plugin does not support underlying moogose calls to MongoDB

### ISSUE THREAD

**Issue by CyberHippo** [open-telemetry/opentelemetry-js-contrib#217]
### What version of OpenTelemetry are you using?
```json
    "@opentelemetry/core": "^0.11.0",
    "@opentelemetry/exporter-jaeger": "^0.11.0",
    "@opentelemetry/node": "^0.11.0",
    "@opentelemetry/plugin-express": "^0.10.0",
    "@opentelemetry/plugin-http": "^0.11.0",
    "@opentelemetry/plugin-https": "^0.11.0",
    "@opentelemetry/tracing": "^0.11.0",
    "@opentelemetry/plugin-mongodb": "^0.10.0",
```

### What version of Node are you using?

```sh
$ node -v
v12.18.3
```

### What did you do?

I enabled the MongoDB plugin for my project as follows : 
```js
const { NodeTracerProvider } = require('@opentelemetry/node');

const provider = new NodeTracerProvider({
  plugins: {
    mongodb: {
      enabled: true,
      path: '@opentelemetry/plugin-mongodb',
    }
  }
});
```
My project uses Mongoose but the plugin does not seems able to properly instrument the underlying calls to [mongod](https://github.com/mongodb/node-mongodb-native) made by Mongoose. 

I have seen [this issue](https://github.com/open-telemetry/opentelemetry-js-contrib/issues/18) and for now I am using the [mongoose plugin from wdalmut](https://github.com/wdalmut/opentelemetry-plugin-mongoose) which works fine and create traces.

Is there any plan to support this feature in the future ? 

### What did you expect to see?

I expect traces to show up.

### What did you see instead?

No traces are being created.

---

PS : Thanks for all the great work being done here!

**Comment by vmarchaud** [comment]
Could you share which mongoose you are using ?

**Comment by CyberHippo** [comment]
@vmarchaud Sure, we are using the following version : 
```json
"mongoose": "^5.9.2",
```

**Comment by wdalmut** [comment]
This should be solved by PR #219
